src/Tribe/Admin/Notices.php

```diff
@@ -557,11 +557,11 @@ public function undismiss_for_all( $slug ) {
 	 *
 	 * @since 4.3
 	 *
-	 * @param string          $slug             Slug to save the notice
-	 * @param callable|string $callback         A callable Method/Function to actually display the notice
-	 * @param array           $arguments        Arguments to Setup a notice
-	 * @param callable|null   $active_callback  An optional callback that should return bool values
-	 *                                          to indicate whether the notice should display or not.
+	 * @param string              $slug             Slug to save the notice
+	 * @param callable|string     $callback         A callable Method/Function to actually display the notice
+	 * @param array|object|string $arguments        Arguments to Setup a notice, must be in a format that wp_parse_args can use.
```

Comment:
Fixes "Argument '3' passed to tribe_notice() is expected to be of type array, string given" warning.